src/components/cards/CreatePool/TokenPrices.vue

```diff
@@ -0,0 +1,112 @@
+<script lang="ts" setup>
+import usePoolCreation from '@/composables/pools/usePoolCreation';
+import useBreakpoints from '@/composables/useBreakpoints';
+import useNumbers from '@/composables/useNumbers';
+import useTokens from '@/composables/useTokens';
+
+import { computed } from 'vue';
+
+type Props = {
+  toggleUnknownPriceModal: () => void;
+};
+
+defineProps<Props>();
+
+/**
+ * COMPOSABLES
+ */
+const { upToLargeBreakpoint } = useBreakpoints();
+const { tokensList } = usePoolCreation();
+const { tokens, priceFor, injectedPrices } = useTokens();
+const { fNum } = useNumbers();
+
+/**
+ * COMPUTED
+ */
+const validTokens = computed(() => tokensList.value.filter(t => t !== ''));
+const knownTokens = computed(() =>
+  validTokens.value.filter(t => priceFor(t) !== 0 && !injectedPrices.value[t])
+);
+const unknownTokens = computed(() =>
+  validTokens.value.filter(t => priceFor(t) === 0 || injectedPrices.value[t])
+);
+const hasUnknownPrice = computed(() =>
+  validTokens.value.some(t => priceFor(t) === 0)
```

Comment:
I know its a bit annoying, but its easier to read `token` instead of `t` for the array filters.

Reply:
yeah good point